ch1-arraysAndStrings/wmassey/README.md

```diff
@@ -0,0 +1,72 @@
+# Cracking the Coding Interview
+##  🔔 Background Info 
+
+My Coding solutions for _Cracking the Coding Interview_, chapter 01: _Arrays and Strings_
+
+---
+## Technology Framework 
+🚀 This project was built with  
+
+
+![made-with-VS-Code](https://img.shields.io/badge/1.60.0-blue?&labelColor=black&label=VS%20COde&logo=visualstudiocode&logoColor=white)
+
+![made-with-Python](https://img.shields.io/badge/3.9.6-blue?&labelColor=black&label=Python&logo=python&logoColor=white)
+
+---
+
+
+## Requirements
+✔️ This project requires the following installations
+
+![made-with-Python](https://img.shields.io/badge/3.9.6-blue?&labelColor=black&label=Python&logo=python&logoColor=white)
+
+---
+## Content
+
+|Problem #| Problem Title        | Stats |
+|---------|----------------------|-------|
+| 1-1     |is_unique             |  ✅  |
+| 1-2     |check_permutation     |  ✅  |
+| 1-3     |URLify                |  ✅  |
+| 1-4     |palindrome_permutation|  🤬  |
+| 1-5     |one_way               |  ❌  |
+| 1-6     |string_compression    |  ✅  |
+| 1-7     |rotate_matrix         |  ❌  |
+| 1-8     |zero_matrix           |  ❌  |
+| 1-9     |string_rotation       |  ✅  |
+
+
+
+---
+## 🤝 Contribute
+ 
+This project is Open Source. Feedback, Questions and [**Pull Requests**](https://github.com/seraph776/REPONAME/pulls) are welcome. 
+For major changes, please open an [**Issue**](https://github.com/seraph776/python-random-quote/issues) first to discuss what you would like to change.
+Please make sure to update tests as appropriate.
+  
+
+
+
+---
+## Contact me 
+🌎 If you have any questions about this project, feel free to get in touch.
+
+
+[![Gmail](https://img.shields.io/badge/-blue?&labelColor=black&label=Gmail&logo=gmail&logoColor=white)](mailto:seraph776)
+[![Twitter](https://img.shields.io/badge/-blue?&labelColor=black&label=Twitter&logo=Twitter&logoColor=white)](https://twitter.com/seraph776) 
+[![GitHub](https://img.shields.io/badge/-blue?&labelColor=black&label=GitHub&logo=github&logoColor=white)](https://github.com/seraph776) 
+[![LinkedIn](https://img.shields.io/badge/-blue?&labelColor=black&label=linkedin&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/wmassey776/)
+
+
+
+---
+## License  
```

Comment:
Please remove this as well.

Reply:
All taken care of! Let me know if there is anything else I should do - Thanks again!